<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="Dashboard" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qcDrive</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/dashboard.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>961</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="IndexWidget" name="index_widget"/>
   <widget class="Mp3Window" name="mp3_widget"/>
   <widget class="MainWindow" name="browser_widget"/>
   <widget class="MapB" name="maps_widget"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>66</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="btn_group">
    <item>
     <widget class="QPushButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0); color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0); color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn3">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0); color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0); color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn5">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0); color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btn5</zorder>
   <zorder>btn3</zorder>
   <zorder>btn4</zorder>
   <zorder>btn2</zorder>
   <zorder>btn1</zorder>
  </widget>
  <widget class="QLabel" name="bgc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 88, 88);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="miniWeather" name="mini_weather" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <zorder>bgc</zorder>
  <zorder>bg</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>mini_weather</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IndexWidget</class>
   <extends>QWidget</extends>
   <header>index.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Dashboard</class>
   <extends>QWidget</extends>
   <header>dashboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mp3Window</class>
   <extends>QWidget</extends>
   <header>subprojects/mp3/mp3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWindow</class>
   <extends>QWidget</extends>
   <header>subprojects/myQWebEngine/mainwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapB</class>
   <extends>QWidget</extends>
   <header>subprojects/MapB/mapb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>miniWeather</class>
   <extends>QWidget</extends>
   <header>miniWeather.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
